test/cluster/mzcompose.py

```diff
@@ -2546,3 +2547,68 @@ def worker(c: Composition, i: int) -> None:
         assert (
             p99 < p99_limit
         ), f"p99 is {p99:.2f}s, should be less than {p99_limit:.2f}s"
+
+
+def workflow_test_incident_70(c: Composition) -> None:
+    """
+    Test incident-70.
+    """
+    num_conns = 10
+    persist_reader_lease_duration_in_sec = 30
+    data_scale_factor = 10
+
+    c.down(destroy_volumes=True)
+    c.up("materialized")
+
+    c.sql(
+        f"ALTER SYSTEM SET max_connections = {num_conns + 5};",
+        port=6877,
+        user="mz_system",
+    )
+
+    c.sql(
+        f"ALTER SYSTEM SET persist_reader_lease_duration = '{persist_reader_lease_duration_in_sec}s';",
+        port=6877,
+        user="mz_system",
+    )
+
+    c.sql(
+        f"""
+              CREATE SOURCE gen FROM LOAD GENERATOR TPCH (SCALE FACTOR {data_scale_factor}) FOR ALL TABLES;
```

Comment:
Do you need to put load on the source cluster? If so, you need to specify a very small TICK INTERVAL, otherwise not much is happening with the default TICK INTERVAL which is 1 second I believe.

Reply:
Thanks, I will try